### PR TITLE
Style windows and use Status colors

### PR DESCRIPTION
**Pull request by jrainville**
Styles all the windows and adds bootstrap overrides, so that they use the Status colors and styles

image
image

### CODE REVIEW

**Comment by iurimatias** [comment]
@jrainville please rebase & merge

File: .storybook/config.js [hunk 1]
```diff
@@ -24,6 +24,7 @@ configure(loadStories, module);
 
 import 'bootstrap/dist/css/bootstrap.css';
```
Comment (alaibe) [inline .storybook/config.js]: You don't need this one anymore?

File: app/js/components/Header.scss [hunk 2]
```diff
@@ -0,0 +1,8 @@
+header .navbar {
+  margin-bottom: 32px;
+
+  .logo-text {
```
Comment (alaibe) [inline app/js/components/Header.scss]: logo-text => text-logo to keep the same logic if possible

File: app/js/components/Reputation.scss [hunk 3]
```diff
@@ -0,0 +1,33 @@
+.rating-container {
```
Comment (alaibe) [inline app/js/components/Reputation.scss]: What about keeping the container name the same as the component:
e.g => reputation (or feel free to change the name of the component)

File: app/js/components/SellerInformation.jsx [hunk 4]
```diff
@@ -2,29 +2,29 @@ import React from 'react';
...
   address: PropTypes.string,
   name: PropTypes.string,
-  isPositiveRating: PropTypes.bool,
-  nbTrades: PropTypes.number,
-  type: PropTypes.string
+  reputation: PropTypes.object
 };
 
-export default ProfileInformation;
+export default SellerInformation;
```
Comment (alaibe) [inline app/js/components/SellerInformation.jsx]: :100:

File: app/js/containers/ProfileContainer.jsx [hunk 5]
```diff
@@ -18,10 +17,10 @@ class ProfileContainer extends Component {
 
   render() {
     const profile = this.props.profile;
+    console.log(profile);
```
Comment (alaibe) [inline app/js/containers/ProfileContainer.jsx]: Can be removed

File: app/js/containers/Seller/4_SellerMarginContainer.jsx [hunk 6]
```diff
@@ -39,7 +37,9 @@ class SellerMarginContainer extends Component {
   marginChange = (margin) => {
     margin = parseInt(margin, 10);
     this.validate(margin);
-    this.setState({margin});
+    if (!isNaN(margin)) {
```
Comment (alaibe) [inline app/js/containers/Seller/4_SellerMarginContainer.jsx]: :100: I just found the bug 2min ago

File: app/js/index.js [hunk 7]
```diff
@@ -7,6 +7,7 @@ import i18n from './i18n';
 
 import 'bootstrap/dist/css/bootstrap.css';
```
Comment (alaibe) [inline app/js/index.js]: you don't need this one anymore?

**Pull request merged**